Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationCode.bind

```diff
@@ -26,4 +26,28 @@
         public string DeviceManufacturer => SystemInformation.DeviceManufacturer;
 
         // To get available memory in MB
-        public float AvailableMemory => SystemInformation.AvailableMemory;
\ No newline at end of file
+        public float AvailableMemory => SystemInformation.AvailableMemory;
+
+        // To get if the app is being used for the first time since it was installed.
+        public string IsFirstUse => SystemInformation.IsFirstRun.ToString();
```

Comment:
I don't think we need the ToString on all the properties in the bind file. It hides the return type of the property/function.

Reply:
done